.env.example

```diff
@@ -29,10 +28,11 @@ REACT_APP_FOREIGN_GAS_PRICE_FALLBACK=5000000000
 REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL=15000
 
 # Default
-REACT_APP_DESCRIPTION="The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner."
+REACT_APP_TITLE=TokenBridge UI app - %c
+REACT_APP_DESCRIPTION=The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner.
 
 # RSK
-#REACT_APP_DESCRIPTION="The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network to Rootstock network in a quick and cost-efficient manner."
+#REACT_APP_DESCRIPTION=The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network to Rootstock network in a quick and cost-efficient manner.
```

Comment:
The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network and the Rootstock network in a quick and cost-efficient manner.

Reply:
thanks! addressed by https://github.com/poanetwork/bridge-ui/pull/203/commits/ca343bf57cedee155e489f7fe229c6e22c8fbfb5